### ISSUE TITLE
Output predicted race probabilities for functions in ethnicolr2

### ISSUE THREAD

**Issue by jcweaver** [appeler/ethnicolr2#1]
Functions in ethnicolr original package produce columns containing the percent chance white, Black, Asian, Hispanic. These columns are not produced in functions using ethnicolr2.

**Comment by soodoku** [comment]
@rajashekar, let's look into adding softprob. for each class along with the predicted labels.

**Comment by rajashekar** [comment]
@jcweaver and @soodoku  -  This is fixed in version `0.1.1`, please get the update version. Added another column `probs`. Please verify. 

Sample output - 
```
        last   first  ...     preds                                                                                                                    probs
0  hernandez  hector  ...  hispanic  {'asian': 0.0003889082, 'hispanic': 0.987706, 'nh_black': 0.0012073584, 'nh_white': 0.008855153, 'other': 0.0018425211}
1      zhang   simon  ...     asian   {'asian': 0.936533, 'hispanic': 0.0016647797, 'nh_black': 0.0012903506, 'nh_white': 0.011221227, 'other': 0.049290624}
```

**Comment by soodoku** [comment]
thanks @rajashekar.
